src/components/RadioBorderedBox/index.tsx

```diff
@@ -37,55 +39,61 @@ const StyledBorderedBox = styled(BorderedBox)<BorderedBoxTypes>`
     if (error)
       return `
         border: 1px solid ${colors.danger.borderWeak} !important;
-        box-shadow: ${shadows.focusDanger};
       `
 
     if (checked)
       return `
         border: 1px solid ${colors.primary.borderWeak} !important;
-        box-shadow: ${shadows.focusPrimary};
       `
 
     return null
-  }}
+  }};
 `
 
 const StyledRadioContainer = styled.div`
   display: flex;
   align-items: center;
   gap: 0 ${({ theme }) => theme.space['1']};
-  margin-bottom: ${({ theme }) => theme.space['1']};
+`
+
+const StyledSpaceContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
 `
 
 const StyledError = styled.div`
   font-size: 12px;
   color: ${({ theme }) => theme.colors.danger.text};
-  padding: ${({ theme }) => `0 ${theme.space['0.5']}`};
 `
 
-const StyledExpandable = styled(Expandable)`
-  margin-top: ${({ theme }) => theme.space['1']};
-  margin-bottom: ${({ theme }) => theme.space['1']};
+const StyledMainContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${({ theme }) => theme.space['1']};
 `
 
 type RadioBorderedBoxProps = RadioProps & {
   badgeSize?: ComponentProps<typeof Badge>['size']
   badgeText?: string
+  badgeProminence?: ComponentProps<typeof Badge>['prominence']
```

Comment:
we can improve RadioBorderBox->Badge story to add `badgeProminence` in one of the three examples